modules/microshift-firewall-update-for-service.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-firewall.adoc
+
+
+:_content-type: CONCEPT
+[id="microshift-firewall-update-for-service_{context}"]
+= Updating firewall ports when a service is exposed
+
+Firewalld is often active when you run services on {microshift-short}. This can disrupt certain services on {microshift-short} because traffic to the ports might be blocked by the firewall. You need to ensure the necessary firewall ports are open if you want certain services to be accessible from outside the host. There are several options for opening your ports:
```

Comment:
```suggestion
Firewalld is often active when you run services on {microshift-short}. This can disrupt certain services on {microshift-short} because traffic to the ports might be blocked by the firewall. You must ensure that the necessary firewall ports are open if you want certain services to be accessible from outside the host. There are several options for opening your ports:
```